pages/validators/full_node_streaming.md

```diff
@@ -361,4 +376,11 @@ Q: What are the exec modes?
 	ExecModeEndBlock            = 101
 	ExecModePrepareCheckState   = 102
 ```
-</details>
\ No newline at end of file
+</details>
+<br>
+
+Q: I only want to listen to confirmed updates. I do not want to process optimistic fills.
+- A: You will want to only process messages from `MsgProposedOperations` DeliverTx stage. These updates will have exec mode execModeFinalize.
```

Comment:
`MsgProposedOperations` isn't really mentioned in the doc until here. I think it should be replaced with saying they should listen to the `StreamOrderbookFill` stream

Reply:
modified to remove this word